### PR TITLE
update keep safe page

### PR DESCRIPTION
**Pull request by sash-binary**
# Description

Updated Keep Safe page

Changes:

-   new icons
-   reduced number of components
-   added css grid component

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [x] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/images/svg/cloudfare-dns.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 64 64" aria-labelledby="configure-icon-title">
```
Comment (sara-fs) [inline src/images/svg/cloudfare-dns.svg]: Should `role="img"` be added to this line?

File: src/images/svg/demo-account.svg [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 64 64">
```
Comment (sara-fs) [inline src/images/svg/demo-account.svg]: Should `role="img"` be added to this line? and `aria-labelledby`

Comment (sara-fs) [reply src/images/svg/demo-account.svg]: Should    `<title id="....">....</title> ` be added to this svg file?

File: src/images/svg/don-t-share-payment-methods.svg [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 64 64" aria-labelledby="dont-share-icon-title">
```
Comment (sara-fs) [inline src/images/svg/don-t-share-payment-methods.svg]: And the same thing for this line.

File: src/images/svg/install-latest-antivirus.svg [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 64 64" aria-labelledby="install-antivirus-icon-title">
```
Comment (sara-fs) [inline src/images/svg/install-latest-antivirus.svg]: And here.

File: src/images/svg/keep-browser-up-to-date.svg [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="66" viewBox="0 0 64 66" aria-labelledby="browser-icon-title">
```
Comment (sara-fs) [inline src/images/svg/keep-browser-up-to-date.svg]: And here.

File: src/images/svg/recent-logins.svg [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 64 64">
```
Comment (sara-fs) [inline src/images/svg/recent-logins.svg]: And here.

File: src/images/svg/two-factor-authentication.svg [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 64 64" aria-labelledby="two-factor-auth-icon-title">
```
Comment (sara-fs) [inline src/images/svg/two-factor-authentication.svg]: And here.

File: src/pages/keep-safe/_icon-grid.js [hunk 8]
```diff
@@ -5,53 +5,54 @@ import device from 'themes/device'
...
+        }
     }
 `
 
-const Col = ({ Icon, content, grid_name }) => (
-    <GridCol grid_name={grid_name}>
+const Col = ({ Icon, content }) => (
+    <GridCol>
         <Icon />
         <Text color="black-3" lh="1.55">
```
Comment (sara-fs) [inline src/pages/keep-safe/_icon-grid.js]: `color="black-3"` is the default value. We can remove it.

File: src/pages/keep-safe/index.js [hunk 9]
```diff
@@ -164,110 +55,83 @@ const KeepSafe = () => {
...
-                                <Text color="black-3">
-                                    {localize(
-                                        'Trading on Deriv.com can become addictive. If you wish to stop trading for a period of time, please use our self-exclusion facilities.',
-                                    )}
+                            </CssGridColumn>
+                            <CssGridColumn>
+                                <CheckedIcon />
+                            </CssGridColumn>
+                            <CssGridColumn>
+                                <Text color="black-3" lh="1.55" margin="0.3rem 0 0">
```
Comment (sara-fs) [inline src/pages/keep-safe/index.js]: color="black-3"  can be removed. It's the default value.

File: src/pages/keep-safe/index.js [hunk 10]
```diff
@@ -1,156 +1,46 @@
...
-`
-
-const GridSectionContainer = styled(SectionContainer)`
-    grid-area: practice;
-    padding: 8rem 0;
-
-    @media ${device.tabletL} {
-        padding: 4rem 0;
-    }
+const Cta = styled(Button)`
```
Comment (oskar-binary) [inline src/pages/keep-safe/index.js]: The normal button styling is not as per Zeplin?

Comment (sash-binary) [reply src/pages/keep-safe/index.js]: No, its larger (padding, border radius)

Comment (oskar-binary) [reply src/pages/keep-safe/index.js]: font size and line height can be removed?

**Pull request merged**